website/integrations/services/cloudflare-access/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Integrate with Cloudflare Access
+sidebar_label: Cloudflare Access
+---
+
+# Cloudflare Access
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Cloudflare Access
+
+> Cloudflare Access is a secure, cloud-based zero-trust solution for managing and authenticating user access to internal applications and resources.
+>
+> -- https://www.cloudflare.com/zero-trust/products/access/
+
+## Preparation
+
+The following placeholders are used in this guide:
+
+- `company.cloudflareaccess.com` is the FQDN of your Cloudflare Access subdomain.
+- `authentik.company` is the FQDN of the authentik install.
+
+To proceed, you need to register for a free Cloudflare Access account and have both a Cloudflare account and a publicly accessible authentik instance with a trusted SSL certificate.
+
+## authentik configuration
+
+1. From the Admin interface, navigate to **Applications** -> **Applications**.
+2. Use the wizard to create a new application and provider. During this process:
```

Comment:
For CF-access we can;t use the generic templte here; there are too many details about the configuration settings that we need to provide. Let's use the more robust structure, with space to call out specific values needed for both the app and the provider. For example, for the app, they need to need to set the **Policy engine mode** is set to **ANY**. There are a few specific config settings for providers as well (client type to **Confidential**). Take a very careful look at each of the steps, each of the values, called out in the original guide, and make sure they don't get lost, please. ;-)

Reply:
Agree that if they are defaults (and I like your sentence telling them they can leave defaults) AND if they are not specifically called out in the original guide, then we can skip them. 

But any of the values that are explicitly stated or called out in the CF guide need to be called out explicitly in this new Guide too.. even if it is the default. This is one that Jens and I stepped through, so I am confident that we need to mention those specific values. Let's try it see how it look, please.